### PR TITLE
open source site redesign

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

open source site redesign

## What is the current behavior?

CleanShot 2023-05-30 at 14 06 04@2x

## What is the new behavior?

CleanShot 2023-05-30 at 14 05 55@2x

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/www/pages/open-source/index.tsx [hunk 1]
```diff
@@ -56,103 +93,119 @@ const OpenSource = () => {
...
-                      >
-                        <div className="grow bg-gray-300 dark:bg-gray-400 p-4 rounded-lg">
-                          <div className="flex gap-4 shrink-0">
+        <SectionContainer>
+          <div className="text-center space-y-2">
+            <h2 className="text-2xl">Community Maintainers</h2>
+            <p className="w-[50%] mx-auto text-scale-1100 text-sm pb-8">
+              Thanks to our dedicated community maintainers, we're able to propel our open source
+              mission forward. Their unwavering commitment drives our development, nurtures
+              collaboration, and empowers us to forge a more inclusive and accessible future.
```
Comment (Isaiah-Hamilton) [inline apps/www/pages/open-source/index.tsx:153]: Kind of winged it for this so we might want to change it, unless its fine the way it is.

File: apps/www/pages/open-source/index.tsx [hunk 2]
```diff
@@ -56,103 +93,121 @@ const OpenSource = () => {
...
+              Thanks to our sponsors we're able to advance our open source mission. Their support
+              fuels our development, fosters collaboration, and empowers us to shape a more open and
+              accessible future.
+            </p>
+            <div>
+              {tiers.map(
+                (t: any, i: number) =>
+                  !!t.transactions.length && (
+                    <div key={i} className="mt-8">
+                      <h3 className="font-medium mb-6">{t.heading}</h3>
```
Comment (fsansalvadore) [inline apps/www/pages/open-source/index.tsx:124]:
```suggestion
                      <h3 className="font-mono text-scale-900 mb-6">{t.heading}</h3>
```
Based on Figma this should be mono and slightly darker.
Schermata 2023-06-01 alle 12 29 02

**Review by fsansalvadore — CHANGES_REQUESTED** [review]
Great job, just added a small suggestion 👍

File: apps/www/pages/open-source/index.tsx (hunk 1 above)
Comment (fsansalvadore) [reply apps/www/pages/open-source/index.tsx:155]: Sounds good

**Comment by awalias** [comment]
@Isaiah-Hamilton looks like this one has been superceded? 
image

**Pull request closed without merging**